crates/experimentation_client/src/interface.rs

```diff
@@ -187,9 +187,11 @@ pub extern "C" fn get_applicable_variant(
     let variants = local.block_on(&Runtime::new().unwrap(), unsafe {
```

Comment:
Can we remove this line ?
  // println!("Fetching variantIds");

Reply:
It's already removed, you seeing the code on the opposite(removed) side.